Assignment1/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,14 @@
+eclipse.preferences.version=1
```

Comment:
Please don't commit IDE configs, generated code and binary files in git.

Reply:
okay.